eaf.el

```diff
@@ -932,13 +931,20 @@ This should be used after setting `eaf-browser-continue-where-left-off' to t."
                                 (split-string (buffer-string) "\n" t))))
         (if (epc:live-p eaf-epc-process)
             (dolist (url browser-url-list)
-              (eaf-open-browser url))
+              (if (eaf-is-valid-web-url url)
+                  (eaf-open-browser url)
+                (eaf-open url)))
           (dolist (url browser-url-list)
-            (push `(,url "browser" "") eaf--active-buffers))
-          (when eaf--active-buffers (eaf-open-browser (nth 0 (car eaf--active-buffers))))))
+            (if (eaf-is-valid-web-url url)
+                (push `(,url "browser" "") eaf--active-buffers)
+              (push `(,url "pdf-viewer" "") eaf--active-buffers)))
```

Comment:
eaf-pdf-viewer shouldn't be hard-coded here. It is understandable that the Browser is a much more unique feature than other EAF Applications that work with files, but it is weird that only PDF viewers can be restored but not other things such as Video players etc.

Reply:
Try to make it more general and work for other EAF applications.